tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -30,6 +30,7 @@ type CustomClusterProvisioningTestSuite struct {
 	client             *rancher.Client
 	session            *session.Session
 	standardUserClient *rancher.Client
+	provisioning       *provisioning.Config
```

Comment:
@jameson-mcghee sorry, I did not notice this yesterday. Since 2.6 does not have the hardening changes that 2.7 does yet for RKE2, you can remove this line and line 53.

Reply:
Done